### DISCUSSION TITLE
Genome scale modelling FBA example

### DISCUSSION THREAD

**Discussion by abshah1** [opencobra/cobrapy#1052] [General]
Hi,
I'm creating a model of a metabolic network for my Uni project. I need to use FBA and MFA, and I was wondering if anyone has FBA and MFA models that I could use to build my own. Any help would be greatly appreciated. 
Thanks

**Comment by Midnighter** [comment]
Hello @abshah1,

For which organism do you want to build a model? Maybe one exists already? There are multiple model databases out there by now. Other than that [our documentation](cobrapy.readthedocs.io/) is a great place to start and you could check out [KBase](https://www.kbase.us/) if you really need to create a new reconstruction from a genome.

**Reply by abshah1** [reply]
Hi @Midnighter 

Thank you for your response, I will check those sites out.
I am trying to optimise a metabolic reaction network, that includes the glycolysis pathway, the pentose phosphate pathway, the citric acid cycle, and the 2-C-methyl-D-erythritol 4-phosphate (MEP) pathway, in an attempt to overproduce high value compunds like terpenes. The organism used is cyanobacteria. The method involves applying FBA, MFA and Optknock.

**Reply by Midnighter** [reply]
For OptKnock you should take a look at [cameo](https://cameo.bio/) or maybe you would like to use a visual online tool like https://caffeine.dd-decaf.eu/. I don't know about your specific cyanobacterium but I know that some have been published. Take a look at [BioModels](https://www.ebi.ac.uk/biomodels/), for example.
